config/config_mgmt.py

```diff
@@ -0,0 +1,864 @@
+        '''
+        Read the config in Config DB. Assign it in self.configdbJsonIn.
+
+        Parameters:
+            (void)
+
+        Returns:
+            (void)
+        '''
+        print('Reading data from Redis configDb')
+        # Read from config DB on sonic switch
+        db_kwargs = dict(); data = dict()
+        configdb = ConfigDBConnector(**db_kwargs)
+        configdb.connect()
+        deep_update(data, FormatConverter.db_to_output(configdb.get_config()))
+        self.configdbJsonIn =  FormatConverter.to_serialized(data)
+        self.sysLog(syslog.LOG_DEBUG, 'Reading Input from ConfigDB {}'.\
+            format(self.configdbJsonIn))
+
+        return
+
+    def writeConfigDB(self, jDiff):
+        '''
+        Write the diff in Config DB.
+
+        Parameters:
+            jDiff (dict): config to push in config DB.
+
+        Returns:
+            void
+        '''
+        print('Writing in Config DB')
+        db_kwargs = dict(); data = dict()
+        configdb = ConfigDBConnector(**db_kwargs)
+        configdb.connect(False)
+        deep_update(data, FormatConverter.to_deserialized(jDiff))
+        data = sort_data(data)
+        self.sysLog(msg="Write in DB: {}".format(data))
+        configdb.mod_config(FormatConverter.output_to_db(data))
+
+        return
+
+# End of Class ConfigMgmt
+
+class ConfigMgmtDPB(ConfigMgmt):
+    '''
+        Config MGMT class for Dynamic Port Breakout(DPB). This is derived from
+        ConfigMgmt.
+    '''
+
+    def __init__(self, source="configDB", debug=False, allowTablesWithoutYang=True):
+        '''
+        Initialise the class
+
+        Parameters:
+            source (str): source for input config, default configDb else file.
+            debug (bool): verbose mode.
+            allowTablesWithoutYang (bool): allow tables without yang model in
+                config or not.
+
+        Returns:
+            void
+        '''
+        try:
+            ConfigMgmt.__init__(self, source=source, debug=debug, \
+                allowTablesWithoutYang=allowTablesWithoutYang)
+            self.oidKey = 'ASIC_STATE:SAI_OBJECT_TYPE_PORT:oid:0x'
+
+        except Exception as e:
+            print(e)
+            raise(Exception('ConfigMgmtDPB Class creation failed'))
+
+        return
+
+    def __del__(self):
+        pass
+
+    def _checkKeyinAsicDB(self, key, db):
+        '''
+        Check if a key exists in ASIC DB or not.
+
+        Parameters:
+            db (SonicV2Connector): database.
+            key (str): key in config DB, with table Seperator if applicable.
+
+        Returns:
+            (bool): True, if given key is present.
+        '''
+        self.sysLog(msg='Check Key in Asic DB: {}'.format(key))
+        try:
+            # chk key in ASIC DB
+            if db.exists('ASIC_DB', key):
+                return True
+        except Exception as e:
+            print(e)
+            raise(e)
+
+        return False
+
+    def _testRedisCli(self, key):
```

Comment:
While internal LKDN review, we thought it is a good idea to double check ASICDB in debug mode. This is basically testing _verifyAsicDB function. 
We can removed it, since we have good amount of other tests, which will fail now if ASIC DB have old ports.

Reply:
Thanks. Please remove. Any issue, your tests will catch it.